TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ThumbnailViewHelper.php

```diff
@@ -20,12 +20,12 @@
 use TYPO3\Media\Domain\Service\ThumbnailService;
 
 /**
- * Renders an <img> HTML tag from a given TYPO3.Media's asset instance
+ * Renders an <img> HTML tag from a given Neos.Media's asset instance
  *
  * = Examples =
  *
  * <code title="Rendering an asset thumbnail">
- * <typo3.media:thumbnail asset="{assetObject}" alt="a sample asset without scaling" />
+ * <neos.media:thumbnail asset="{assetObject}" alt="a sample asset without scaling" />
```

Comment:
Yet again: Even though this is just a comment (used to render documentation!), it will lead people to the use of a not (yet) existing ViewHelper. Repeat below. So, please only merge with the actual vendor name change.